docroot/themes/humsci/humsci_basic/src/js/main-nav/nested-toggler.js

```diff
@@ -0,0 +1,72 @@
+const togglers = document.querySelectorAll('.hb-nested-toggler');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+
+if (togglers) {
+  for (let i = 0; i < togglers.length; i += 1) {
+    const toggler = togglers[i];
+    const togglerID = toggler.getAttribute('id');
+    const togglerContent = document.querySelector(`[aria-labelledby="${togglerID}"]`);
+    const togglerParent = toggler.closest('.su-main-nav__item--parent');
+    const shouldBeOpen = togglerParent.classList.contains('su-main-nav__item--active-trail') ? true : false;
+    let isHidden;
+
+    const collapseMenu = () => {
+      togglerContent.setAttribute('aria-hidden', true);
```

Comment:
I am on the fence with the aria-hidden stuff. technically, I think you should never hide a tabbable element (such as the a tags under the ul). I think it would be best to apply display: none; instead. we could add a class and then do that in the styles

Reply:
I think what happens is that even if you set aria-hidden to true, you can still tab around, but nothing will be said, which is not a great experience, and can be very confusing for screen reader users

Reply:
oh i see.. you do set it to display:none;  still feels like you could add a class instead of aria-hidden since it's not doing much  when the display CSS is used

Reply:
I prefer to use aria-hidden and then target that attribute with the CSS because it helps prevent against future changes breaking things. If we rely on a class and something happens where the class is being toggled but aria attributes are not we probably won't notice right away.